stored_queries/ncbi_tax/ncbi_taxon_get_associated_ws_objects.yaml

```diff
@@ -23,6 +23,16 @@ params:
     ts:
       type: integer
       title: Versioning timestamp
+    select_obj:
+      type: array
+      items: {type: string}
+      title: WS obj fields to keep in the results
+      default: []
+    select_edge:
```

Comment:
I'm still pretty queasy about directly exposing database field names, but this isn't my show...